bundle/config/mutator/normalize_paths.go

```diff
@@ -84,6 +85,19 @@ func collectGitSourcePaths(b *bundle.Bundle) []dyn.Path {
 }
 
 func normalizePath(path string, location dyn.Location, bundleRootPath string) (string, error) {
+	// Handle requirements file paths with -r flag
+	reqPath, ok := strings.CutPrefix(path, "-r ")
```

Comment:
should we also TrimSpace in case there multiple spaces?

Reply:
Good point, fixed + added test

Reply:
What about `--requirement`